developer/js/index.ts

```diff
@@ -221,6 +229,11 @@ export default class LexicalModelCompiler {
       }
     }
 
+    // Figure out the punctuation used in the model.
+    let punctuation: LexicalModelPunctuation = Object.assign(
```

Comment:
As noted, I think it's better if this is done in the LMLayer at runtime?

Reply:
Done in the latest commit!

Reply:
This still seems to be in the compiler?